projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.spec.ts

```diff
@@ -579,29 +580,11 @@ describe('QueryCheckYourAnswersComponent', () => {
   describe('searchAndCompleteTask', () => {
     it('should call search task', () => {
       component.queryCreateContext = QueryCreateContext.NEW_QUERY;
-      component.searchAndCompleteTask();
+      // component.searchAndCompleteTask();
```

Comment:
@olusegz07 minor comment; if this above commented code is not required then can this be remove.